SWLOR.Game.Server/Feature/RecipeDefinition/FabricationRecipeDefinition/FurnitureRecipes.cs

```diff
@@ -1674,20 +2052,43 @@ private void Tier3()
                 .Component("ancient_wood", 6)
                 .Component("lth_good", 3);
 
-            // DNA Extractor III
-            _builder.Create(RecipeType.DNAExtractor3, SkillType.Fabrication)
```

Comment:
Looks like the DNA extractor got deleted in your copy. Make sure you re-add this to your copy.

Reply:
It's down in line 2078

Tried to keep all the DNA extractors at the end of their respected tier for future proofing.

Reply:
Ok, the recipes are in order of level generally speaking (might be a few exceptions) but that's fine. Just wanted to make sure it didn't get deleted. I see it now thanks